server/test/integration/com/thoughtworks/go/config/GoConfigMigrationIntegrationTest.java

```diff
@@ -1462,6 +1462,292 @@ public void shouldCreateProfilesFromMultipleAgentConfigsAcrossPipelines_asPartOf
         assertThat(profiles.get(1), is(expectedAWSProfile));
     }
 
+    @Test
+    public void shouldCreateAViewElementInEachTemplateIfPipelineGroupAdminsExist_asPartOfMigration89() throws NoSuchMethodException, IllegalAccessException, InvocationTargetException {
```

Comment:
To be extremely technical and pedantic, only pipeline group admins who had pipelines depending on templates could see those templates. Right? Or at least, because of an issue where permissions were not properly checked, anyone who knew a template name could view it. Now, all group admins can view all templates by default, unless they're removed. Right?